### PR TITLE
Feat/fill detail

### PR DESCRIPTION
**Pull request by jerometana**
- Improve UI
- Add toast
- Make share button works

# Before

image 

# After

image

### CODE REVIEW

File: layouts/craft-content.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <div>
         <h2>Craft your content</h2>
         <div>
-            <div class="flex flex-col gap-2">
+            <div class="flex flex-col gap-2 ">
```
Comment (Nine0512) [inline layouts/craft-content.vue]: Why?

File: pages/class/[id]/edit.vue [hunk 2]
```diff
@@ -1,24 +1,45 @@
 <script setup lang="ts">
+// const { $useToast } = useNuxtApp()
+const toast = useToast();
+
+const show = () => {
+    toast.add({
+        severity: "secondary",
+        summary: "Url is copied to clipboard",
+        // detail: "Message Content",
```
Comment (Nine0512) [inline pages/class/[id]/edit.vue]: Are these code comments necessary? If not please remove it.

Comment (jerometana) [reply pages/class/[id]/edit.vue]: ครับท่าน

File: pages/class/[id]/edit.vue [hunk 3]
```diff
@@ -1,24 +1,44 @@
 <script setup lang="ts">
+// const { $useToast } = useNuxtApp()
```
Comment (Nine0512) [inline pages/class/[id]/edit.vue]: You forgot this one.

File: pages/class/[id]/edit.vue [hunk 4]
```diff
@@ -1,24 +1,44 @@
...
+const toast = useToast();
+
+const show = () => {
+    toast.add({
+        severity: "secondary",
+        summary: "Url is copied to clipboard",
+        group: "tc",
+        life: 1000,
+    });
+    console.log("show");
```
Comment (Nine0512) [inline pages/class/[id]/edit.vue]: Do we need this one?

**Review by Nine0512 — APPROVED** [review]

**Pull request merged**